app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,12 @@
+<h3><%= _("Puppet Certificate Authority") %></h3>
+<%= show_feature_version('puppetca') %>
+<div class="col-md-4"><%= _('Hosts managed') %></div>
+<div class="col-md-8">
+  <%= Host::Managed.where(:puppet_ca_proxy_id => @smart_proxy.id).count %>
+</div>
+<% if authorized_via_my_scope(:smart_proxy, :view_smart_proxies_puppet_ca, @smart_proxy) %>
+  <div class="col-md-12" data-ajax-url="<%= smart_proxy_puppetca_index_path(:smart_proxy_id => @smart_proxy) %>" data-on-complete="onContentLoad"><%= spinner %></div>
```

Comment:
I find it a bit confusing when its in one tab, mostly the fact the table has no fixed-size, so going over pagination is making my page "jump" around.